pulp_2to3_migration/app/migration.py

```diff
@@ -120,12 +111,11 @@
     """
     # gather all needed plugin importer migrators
     importer_migrators = {}
-    plugins_to_migrate = plan.get_plugins()
 
-    for plugin, plugin_migrator in PLUGIN_MIGRATORS.items():
-        if plugin not in plugins_to_migrate:
+    for plugin in plan.get_plugin_plans():
```

Comment:
This pattern is common enough that it might make sense to have a ```with_migrator=True``` kwarg on this method that just filters out plugins that don't include migrators, agree?

Reply:
I'm not sure when we can do anything without a migrator for a plugin being present, so it might worth returning only plugins with a migrator.

Reply:
Awesome, I will make that change.  I noticed one place where ```get_plugin_plans()``` was being used without doing that check, but I just realized that the rest of the code is using the migrator so that check should be there but is missing.